packages/loader/container-loader/package.json

```diff
@@ -11,6 +11,68 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/index.d.mts",
+				"default": "./lib/index.mjs"
+			},
+			"require": {
+				"types": "./dist/index.d.ts",
+				"default": "./dist/index.cjs"
+			}
+		},
+		"./test/container": {
```

Comment:
Can we file tasks to come back and clean up these packages where we've had to expose test files this way?

Reply:
Did you have a specific remediation in mind? As far as I know this is the only way to export stuff now.

Reply:
If we added an `index.ts` file under `test`, we could just use that single point, rather than a bunch of files. But longer term, packages shouldn't be using other package' test code. We should reorganize if this code is needed by multiple packages IMO.